word_guess.rb

```diff
@@ -0,0 +1,157 @@
+# pair_programming https://github.com/Ada-C7/Word-Guess
+# Elizabeth and Tamiko
+
+require 'colorize'
+
+# new class for word game
+class WordGame
+
+  def initialize
+    @win = false
+    @word = ["PIZZA", "JUMPY", "HELLO", "WORLD", "QUACK", "TAXES"].sample
+    @all_guesses = []
+    @duck_count = 5
+    @open_game = open_game
+
+    # first turn
```

Comment:
Do you need to have a separate section for the first turn here? What does that accomplish?